src/pages/home/ReportScreen.js

```diff
@@ -87,7 +90,7 @@ function getReportID(route) {
     return Number.parseInt(params.reportID, 10);
 }
 
-class ReportScreen extends React.Component {
+class ReportScreen extends React.PureComponent {
```

Comment:
I wanna know why you've made this a PureComponent.

We should only bother with PureComponent if we're noticing performance problems and have determined the reason of unnecessary re-renders. 
`Performance.diffObject()` in `componentDidUpdate()` should help you determine this. 
ref: https://github.com/Expensify/App/blob/main/PERFORMANCE.md#reconciliation

Reply:
> Note: By default, React Navigation applies optimizations to screen components to prevent unnecessary renders. Using a render callback removes those optimizations. So if you use a render callback, you'll need to ensure that you use [React.memo](https://reactjs.org/docs/react-api.html#reactmemo) or [React.PureComponent](https://reactjs.org/docs/react-api.html#reactpurecomponent) for your screen components to avoid performance issues.

Reply:
@mdneyazahmad I don't find that satisfactory.

Again, we should only bother with PureComponent if we're noticing performance problems and have determined the reason of unnecessary re-renders. 

- Does PureComponent reduce number of re-renders? Please provide some kind of verification for this
- PureComponent only does a shallow comparisor, so what side-effects (if any) are expected? https://60devs.com/pure-component-in-react.html

Reply:
@Beamanator can you please buddy check what I've said above https://github.com/Expensify/App/pull/8227#discussion_r835420438

Reply:
I will remove `PureComponent` here.